next/components/forms/info-components/Alert.tsx

```diff
@@ -111,15 +111,15 @@ const Alert = ({
       </div>
       {message && title && (
         <div
-          className={cx('text-p2 w-full pl-9 font-normal', {
+          className={cx('text-p2 w-full font-normal md:pl-9', {
             'text-gray-0': solid,
             'text-gray-700': !solid,
           })}
         >
           {title && message}
         </div>
       )}
-      <AlertButtons className="pl-9" buttons={buttons} />
+      <AlertButtons className="md:pl-9" buttons={buttons} />
     </div>
```

Comment:
I see where this is coming from but that's actually an issue with out figma, the component has it correclty, it's wrong in the designs. 

I've contanted relevant people to fix the figma side. Please revert the changes here.

Reply:
Reverted